gui.py

```diff
@@ -1052,7 +1052,7 @@ def get_title(self, drop: TimedDrop | None) -> str:
             f"{campaign.game.name}\n"
             f"{drop.rewards_text()} "
             f"{drop.progress:.1%} ({campaign.claimed_drops}/{campaign.total_drops})"
-        )
+        )[:128] # string too long Err (maximum length 128)
```

Comment:
Never heard of this being a problem, but if it is, please open an issue for it. Blindly shortening the string isn't a solution I'd want to accept.